core/storage/base_model/gae_models_test.py

```diff
@@ -136,7 +136,6 @@ def test_generic_query_put_get_and_delete_operations(self) -> None:
         model.delete()
         all_models = base_models.BaseModel.get_all()
         self.assertEqual(all_models.count(), 0)
-        self.assertEqual(model_id, 4)
```

Comment:
This seems to be added a long time ago, have you looked into why it was added and why this is causing issues only now?

Reply:
@kevintab95 Also, in this test function is intended to test the creation, retrieval and deletion of models therefore testing the model id seems unnecessary. Moreover, if we want to test the model id, we must explicitly define the id at the time of model creation, which is not the case here.